tests/fracs/test_fracture_importer.py

```diff
@@ -0,0 +1,321 @@
+"""Testing functionality related to the fracture importer. These functions are covered:
+- network_2d_from_csv
+- network_3d_from_csv
+- elliptic_network_3d_from_csv
+
+Created on Wed Dec 12 09:05:31 2018
+
+@author: eke001
+"""
+
+from pathlib import Path
+from typing import Callable
+
+import numpy as np
+import pytest
+
+import porepy as pp
+from porepy.applications.test_utils.arrays import compare_arrays
+from porepy.fracs import fracture_importer
+from porepy.fracs.fracture_network_3d import FractureNetwork3d
+
+# ---------- Testing network_2d_from_csv ----------
```

Comment:
Consider grouping it into a class dedicated to 2d cases.

Reply:
I think we are beyond that point of level now. If it is possible to make guidelines for when to group tests, we could consider adding this to the user guide in the future. We should at least mention the possibility.